components/02-form-elements/typeahead/_typeahead.scss

```diff
@@ -0,0 +1,85 @@
+.typeahead {
+  display: inline-block;
+  position: relative;
+  width: 100%;
+
+  @include mq(s) {
+    width: $input-width;
+  }
+
+  &__combobox {
+    display: none;
+  }
+
+  &__preview {
+    position: absolute;
+    width: 100%;
+
+    line-height: normal;
+    color: #808080;
+
+    border-color: transparent;
+    pointer-events: none;
+  }
+
+  &__input {
+    background: transparent;
+  }
+
+  &__listbox {
+    position: absolute;
+    width: 100%;
+    margin: 2px 0 0;
+    padding: 0;
+    list-style: none;
+
+    border: 1px solid $color-borders;
+    border-top: 0;
+    border-bottom-left-radius: $input-radius;
+    border-bottom-right-radius: $input-radius;
+
+    &:empty {
+      display: none;
+    }
+  }
+
+  &__option {
+    margin: 0;
+    padding: $input-padding;
+    outline: none;
+
+    cursor: pointer;
+
+    &:not(:last-child) {
+      border-bottom: 1px solid $color-borders;
+    }
+
+    &:nth-child(2n) {
+      background: #eee;
+    }
+
+    &:not(&--no-results):hover,
+    &:not(&--no-results):nth-child(2n):hover,
+    &--focused:not(&--no-results),
+    &--focused:not(&--no-results):nth-child(2n) {
+      background: $color-blue;
+      border-color: $color-blue;
+      color: $color-white;
+    }
+
+    &--no-results {
+      cursor: not-allowed;
+      background: $color-dark-grey;
```

Comment:
Contrast is `5.92:1` which passes at AA not AAA.

Reply:
I've lightened the grey which should hopefully improve the contrast ratio